core-application/src/main/java/gohigher/application/service/ApplicationQueryService.java

```diff
@@ -0,0 +1,44 @@
+package gohigher.application.service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import gohigher.application.Application;
+import gohigher.application.port.in.ApplicationQueryPort;
+import gohigher.application.port.in.CalenderApplicationRequest;
+import gohigher.application.port.in.CalenderApplicationResponse;
+import gohigher.application.port.out.persistence.ApplicationPersistenceQueryPort;
+import gohigher.common.Process;
+import lombok.RequiredArgsConstructor;
+
+@Service
+@Transactional(readOnly = true)
+@RequiredArgsConstructor
+public class ApplicationQueryService implements ApplicationQueryPort {
+
+	private final ApplicationPersistenceQueryPort applicationPersistenceQueryPort;
+
+	@Override
+	public List<CalenderApplicationResponse> findByMonth(CalenderApplicationRequest request) {
+		List<Application> applications = applicationPersistenceQueryPort.findByIdAndMonth(request.getUserId(),
+			request.getYear(), request.getMonth());
+
+		List<CalenderApplicationResponse> calenderApplicationResponses = new ArrayList<>();
+		for (Application application : applications) {
+			addScheduleResponses(application, calenderApplicationResponses);
+		}
+		return calenderApplicationResponses;
+	}
+
+	private void addScheduleResponses(Application application,
+		List<CalenderApplicationResponse> calenderApplicationRespons) {
+		for (Process process : application.getProcesses()) {
+			calenderApplicationRespons.add(
+				new CalenderApplicationResponse(application.getCompanyName(), process.getType().name(),
+					process.getSchedule()));
+		}
+	}
```

Comment:
```suggestion
	private void addScheduleResponse(Application application,
		List<CalenderApplicationResponse> calenderApplicationResponse) {
		for (Process process : application.getProcesses()) {
			calenderApplicationRespons.add(
				new CalenderApplicationResponse(application.getCompanyName(), process.getType().name(),
					process.getSchedule()));
		}
	}
```

Domain -> DTO 변환로직은 DTO 에 두면 어떨까요? 제가 좋아하는 방법입니다. ㅎㅎ 해당 로직을 서비스에 두면 서비스에 검증로직만 많이 차지하여 가독성이 떨이지더라구요!

Reply:
해당 로직은 여러 지원서들이 갖는 프로세스를 각각의 DTO로 변환하는 코드라 DTO로 옮길 수가 없습니다ㅠ
만약 response를 `List<CalenderApplicationResponse>`를 바로 반환하는 것이 아닌 해당 값을 필드로 갖는 새로운 DTO로 감싸서 반환한다면 새로운 DTO에 위의 로직을 넣을 수 있을 것 같은데, 현재 응답값은 List로 바로 반환하는 것이라 위치를 옮기는 것은 힘들것 같습니다ㅠㅠ